biorxiv/corpora_comparison/corpora_comparison.ipynb

```diff
@@ -2779,8 +2799,8 @@
    ],
    "source": [
     "preprint_vs_published = get_term_statistics(\n",
-    "    biorxiv_total_count_df, \n",
-    "    pmc_total_count_df, \n",
+    "    preprint_count_df, \n",
```

Comment:
I assume this is where you over-wrote your figure so they looked the same?

Reply:
Correct I accidentally passed the wrong dataframes for this section of code. Hence the duplication.